.storybook/components/TokenSpecimen/TokenSpecimen.tsx

```diff
@@ -1,4 +1,4 @@
-import clsx from 'clsx';
+import { clsx } from 'clsx';
```

Comment:
Weird. What error gets thrown with default export + cjs builds?

Whether it's named or not shouldn't matter.

Reply:
Same question; why is `clsx` having this issue?

Reply:
@booc0mtaco paired on this and saw that `<Link>` is being used in a cjs build from `remix-utils` which `requires` clsx, which I thought would pull the `main` build of `clsx` [according to its package.json](https://github.com/lukeed/clsx/blob/master/package.json#L8) but instead uses the `module` build. Removing that `module` line in clsx's package json in edu-stack allows it to resolve to the `main` build instead and works fine with the default export, implying this issue could arise from import/require resolution within edu-stack

Reply:
Ahhhhhh, the link to remix-utils might be a clue.

I wonder if remix-utils/Link is importing the ESM version of EDS somehow, which then tries to load the ESM versino of clsx? 🤔

Reply:
remix-utils/Link is using the CJS version, but 
How about building both cjs and esm for remix-utils and using conditional exports for it?: [aka](https://github.com/chanzuckerberg/frontend-libs/pull/51)
This works on edu-stack